api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -6142,3 +6143,22 @@ message SessionRecordingConfigUpdate {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// AccessPathChanged is emitted when access graph detects a change in a access path.
+message AccessPathChanged {
+  // Metadata is a common event metadata
+  Metadata Metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // ChangeID is the id of the change.
+  string ChangeID = 2 [(gogoproto.jsontag) = "change_id"];
+
+  // AffectedResourceID is the name of the affected resource.
+  string AffectedResourceName = 3 [(gogoproto.jsontag) = "affected_resource_name"];
```

Comment:
are we emitting an audit per resource?
Should we also include for `identities`?

Reply:
I'd skip it for now and if we decide to add it add it then. The identities can be tricky to get in some cases.

Reply:
> are we emitting an audit per resource?
👍 I'm also interested in this. I guess logic how to group the changes is non-obvious (by matcher?)

Reply:
I missed that.
I think we should emit one audit event per resource, that's why getting the user can be tricky as one resource modification can by caused by multiple events.